sphinx/ext/doctest.py

```diff
@@ -101,12 +126,32 @@ def run(self):
             # parse doctest-like output comparison flags
             option_strings = self.options['options'].replace(',', ' ').split()
             for option in option_strings:
-                if (option[0] not in '+-' or option[1:] not in
-                        doctest.OPTIONFLAGS_BY_NAME):  # type: ignore
-                    # XXX warn?
+                on_or_off, option_name = option[0], option[1:]
```

Comment:
``on_or_off`` is the name given to ``+/-`` in the doctest documentation, but I agree with you that is not meaningful. I wil use ``prefix``, thanks for the suggestion :-)